### PR TITLE
Update: Hello world wrapper & apps

### PR DESCRIPTION
**Pull request by cbrzn**

### CODE REVIEW

File: todo [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ens to register:
+
+wrap://ens/logger.polytest.eth -> wrap://ens/logger.core.polywrap.eth
```
Comment (Niraj-Kamdar) [inline todo:3]: What do you mean by `logger.core`?

**Review by Niraj-Kamdar — DISMISSED** [review]

File: todo (hunk 1 above)
Comment (cbrzn) [reply todo:3]: I mean to literally register that under the subdomain of `core.polywrap.eth` - The reason is that that's how we named it in core https://github.com/polywrap/monorepo/blob/prealpha/packages/js/core/src/interfaces/core-interface-uris.ts#L5

Comment (Niraj-Kamdar) [reply todo:3]: Why do we need to put it under the core though?

Comment (cbrzn) [reply todo:3]: Not strong opinion on this, I guess it makes sense to have the subdomain core? Let's discuss this next week

**Review by Niraj-Kamdar — APPROVED** [review]

**Pull request merged**